middlewares/auth.middleware.js

```diff
@@ -4,7 +4,7 @@ const isAuth = (req, res, next) => {
   } else {
     res
       .status(401)
-      .json({ msg: "You are not authorized to view this resource." });
+      .send({ msg: "You are not authorized to view this resource." });
```

Comment:
```suggestion
      .send({ errors: [{ msg: "You are not authorized to view this resource." }]});
```
to make it consistent with the other errors to be returned to frontend